lib/elements/rebar_hook.rb

```diff
@@ -0,0 +1,48 @@
+require './db/base'
+require './lib/materials/steel'
+
+module Elements
+  class RebarHook < Materials::Steel
+    MAX_FY = 550.to_f
```

Comment:
is this constant conceptually the same as `Rebar::MAX_YIELD_STRESS`?

Reply:
Yes, but we'll keep this deleted